bot/code_coverage_bot/uploader.py

```diff
@@ -47,41 +47,45 @@ def gcp(repository, revision, report, platform=None, suite=None):
     logger.info("Uploaded {} on {}".format(path, bucket))
 
     # Trigger ingestion on backend
```

Comment:
Do we want to ingest all platform/suite combinations? Or should we do it on demand only when somebody requests? (I guess they will be requested seldomly compared to the full report)

Reply:
If we remove that, we need a mechanism to detect the available platforms & suites in the backend

Reply:
OK. Maybe we should have some kind of "partial" ingestion, but that's for the future.

Reply:
We could do a followup where the list of platforms/suites is simply `PUT` on the backend on `/v2/filters` to update the list of possible combinations.